packages/eslint-config-hostelworld-vue/package.json

```diff
@@ -30,6 +30,7 @@
     "eslint-config-hostelworld": "^1.0.x"
   },
   "devDependencies": {
+    "babel-eslint": "^8.0.3",
```

Comment:
The use of babel-eslint is missing on `eslintrc` file

Reply:
should be fine now (see 738534a)